### PR TITLE
Upgrade rules_docker

### PR DESCRIPTION
**Pull request by meteorcloudy**
To fix build failure with Bazel@HEAD caused by bazelbuild/rules_docker#1533
https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/1539

### CODE REVIEW

**Comment by meteorcloudy** [comment]
FYI @aiuto

File: bazel/repositories.bzl [hunk 1]
```diff
@@ -573,9 +573,9 @@ cc_library(
     _maybe(
         http_archive,
         name = "io_bazel_rules_docker",
-        sha256 = "9ff889216e28c918811b77999257d4ac001c26c1f7c7fb17a79bc28abf74182e",
-        strip_prefix = "rules_docker-0.10.1",
-        urls = ["https://github.com/bazelbuild/rules_docker/archive/v0.10.1.tar.gz"],
+        sha256 = "feb53c560be2f97b7d02b23a1738a3154ba89fe630f09a7a838dcad38731b0b8",
+        strip_prefix = "rules_docker-faaa10a72fa9abde070e2a20d6046e9f9b849e9a",
+        urls = ["https://github.com/bazelbuild/rules_docker/archive/faaa10a72fa9abde070e2a20d6046e9f9b849e9a.tar.gz"],
```
Comment (ensonic) [inline bazel/repositories.bzl:578]: I think we also need to add 
```
load("@io_bazel_rules_docker//repositories:deps.bzl", container_deps = "deps")
container_deps()

load("@io_bazel_rules_docker//repositories:pip_repositories.bzl", "pip_deps")
pip_deps()
```

everywhere where we use this as:
```
load("@cloud_robotics//bazel:repositories.bzl", "cloud_robotics_repositories")
cloud_robotics_repositories()

load("@io_bazel_rules_docker//repositories:repositories.bzl", container_repositories = "repositories")
container_repositories()
```

right?

Comment (meteorcloudy) [reply bazel/repositories.bzl:578]: Yes, I believe so

**Comment by ensonic** [comment]
Thanks a lot!

**Pull request closed without merging**